cg/constants/sample_sources.py

```diff
@@ -27,3 +29,7 @@
     "bone marrow",
     "other",
 )
+
+
+class SourceType(StrEnum):
+    UNKNOWN: str = "unknown"
```

Comment:
I don't think we need this, and it seems from the constants above that the normalised name for unknown is `other`

Reply:
"other" is for other tissues not represented/listed in the order. This is, we have information about the tissue type (source) from the order but non of the accepted fields match the actual tissue type for a sample. This is not the same as not having information about the source at all.

Reply:
Sounds good, I see you're using it as an input parameter for Tomte, that's fine I was unsure about setting it to unknown if it comes from lims since I'm not sure it's supported there 👍